docs/reference/javascript-api-for-office.md

```diff
@@ -1,40 +1,132 @@
 ---
-title: JavaScript API for Office
```

Comment:
I like the look of this page. We should use this as the overview page in the ref repo. Let's collab in the new year to get that working.